stdlib/random.pyi

```diff
@@ -9,15 +9,16 @@ _T = TypeVar("_T")
 
 class Random(_random.Random):
     VERSION: ClassVar[int]
+    gauss_next: float | None  # undocumented
```

Comment:
Do you have a use for this? We have a policy to add undocumented things if users ask for them, but we shouldn't go out of our way to add every undocumented attribute.

Reply:
Yes, it is quite widely used: https://cs.github.com/?q=.gauss_next%20language%3APython&scopeName=All%20repos

I personally remember using in it my fake-data generator project.